pkg/network/ebpf/c/protocols/http/http.h

```diff
@@ -164,6 +164,14 @@ static __always_inline bool http_allow_packet(http_transaction_t *http, struct _
         return false;
     }
 
+    protocol_stack_t *stack = get_protocol_stack(&http->tup);
+    if (!stack) {
+        return false;
+    }
+    if (is_fully_classified(stack) || is_protocol_layer_known(stack, LAYER_ENCRYPTION)) {
+        return false;
+    }
```

Comment:
please add a comment about the changes

Reply:
not sure I follow here, but why don't we need to flush the transaction on rst/fin here?